README.md

```diff
@@ -40,87 +42,92 @@ the [Google Cloud Console][dev-console]. You can start adding snapshots and log-
 
-On **Google Container Engine (GKE)**, you need to explicitly add the `cloud_debugger` OAuth scope when creating the cluster:
+### Container Engine
+
+On **Google Container Engine**, you need to explicitly add the `cloud_debugger` OAuth scope when creating the cluster:
 
 ```
 $ gcloud container clusters create example-cluster-name --scopes https://www.googleapis.com/auth/cloud_debugger
 ```
 
-For **Google Compute Engine instances**, you need to explicitly enable the Debugger API access scope for each instance. When creating a new instance through the GCP web console, you can do this in one of two ways under **Identity and API access**:
-* Use the Compute Engine default service account and select "Allow full access to all Cloud APIs" under Access scopes.
-* Select a service account with the [**Cloud Debugger Agent**][debugger-roles] role, which contains the necessary permissions (or any other role with at least the same permissions). You may need to create one if you don't have one already.
+### Compute Engine
+
+For **Google Compute Engine instances**, you need to explicitly enable the `cloud_debugger` Stackdriver Debugger API access scope for each instance. When creating a new instance through the Google Cloud Platform Console, you can do this under **Identity and API access**: Use the Compute Engine default service account and select "Allow full access to all Cloud APIs" under Access scopes.
+
+To use something other than the Compute Engine default service account see the docs for [Creating and Enabling Service Accounts for Instances][service-account-docs] and the [Running elsewhere](#running-locally-and-elsewhere) section below. The important thing is that the service account you use has the [**Cloud Debugger Agent**][debugger-roles] role.
+
+## Running locally and elsewhere
+
+If your application is running outside of Google Cloud Platform, such as locally, on-premise, or on another cloud provider, you can still use Stackdriver Debugger, provided that you supply a project ID and credentials to the Stackdriver Debugger agent.
 
-You may add the Stackdriver Debugger API access scope to existing Compute instances if they are running as a non-default service account by adding the Cloud Debugger Agent role to the service account. For more information, see the docs for [Creating and Enabling Service Accounts for Instances][service-account-docs].
+### Project ID
 
-## Running elsewhere
+Provide a project ID to the Stackdriver Debugger agent by setting the projectId value in the options object passed to the agent's `start` method, or set the `GCLOUD_PROJECT` environment variable:
 
-If your application is running outside of Google Cloud Platform, such as locally, on-premise, or on another cloud provider, you can still use Stackdriver Debugger.
+On Linux or Mac OS X:
+
+```bash
+export GCLOUD_PROJECT=your-project-id
```

Comment:
In the [trace agent's docs](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs#running-elsewhere), we use the form
```
GCLOUD_PROJECT=particular-future-12345 node myapp.js
```
when specifying how to temporarily set `GCLOUD_PROJCT`.

The way you specified, however, is consistent with the other GCP debugger documentation.  I could be helpful to mention the above syntax provided that it works on all systems.